evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -450,7 +455,8 @@ contract FluxAggregator is AggregatorInterface, Owned {
     external
     onlyOwner()
   {
-    require(availableFunds >= _amount, "insufficient available funds");
+    uint256 oracleReserve = uint256(paymentAmount).mul(oracleCount()).mul(RESERVE_ROUNDS);
```

Comment:
I'm pretty sure that since we're performing the same calculation to get the oracleReserve twice, if we turn that into an internal/private function that will save us a bit of contract size.